index.js

```diff
@@ -3,6 +3,12 @@ import postcss from 'postcss';
 import perfectionist from 'perfectionist';
 import postcssSafeParser from 'postcss-safe-parser';
 
+const SUPPORTED_SCOPES = [
+	'source.css',
+	'source.css.less',
```

Comment:
@sindresorhus I believe it will work with anything there is a Post-CSS plugin for, so I think postcss-less would work perhaps? I'm not sure if the `syntax` param only supports `scss` or if `less` would work too.